homeassistant/components/logger/__init__.py

```diff
@@ -70,6 +73,11 @@ def set_log_levels(logpoints):
     if LOGGER_LOGS in config[DOMAIN]:
         set_log_levels(config[DOMAIN][LOGGER_LOGS])
 
+    if LOGGER_FILTERS in config[DOMAIN]:
+        for key, value in config[DOMAIN][LOGGER_FILTERS].items():
+            logger = logging.getLogger(key)
+            _add_log_filter(logger, value)
```

Comment:
Can you combine all the values and make a single call to `_add_log_filter` in case they have multiple/

Reply:
Nevermind, I misread this. I see what you are doing here 👍